docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,442 @@
+metadata:
 
 .. code-block:: console
 
-   Current node cluster state (term, version) pair is (4, 12)
+    sh$ node$./bin/crate-node repurpose
 
-If you have a choice of nodes on which to run this tool then you should choose
-one with a term that is as large as possible. If there is more than one node
-with the same term, pick the one with the largest version. This information
-identifies the node with the freshest cluster state, which minimizes the quantity
-of data that might be lost. For example, if the first node reports (4, 12) and a
-second node reports (5, 3), then the second node is preferred since its term is larger.
-However if the second node reports (3, 17) then the first node is preferred since its
-term is larger. If the second node reports (4, 10) then it has the same term as the
-first node, but has a smaller version, so the first node is preferred.
+    Found 2 tables (2 shards and 2 table metadata) to clean up.
+    Use -v to see list of paths and tables affected.
+    Node is being repurposed as no-master and no-data. Clean-up of table data
+    is performed.
 
-.. WARNING::
+    Do you want to proceed?
 
-  These steps can lead to arbitrary data loss since the target node may not
-  hold the latest cluster metadata, and this out-of-date metadata may make
-  it impossible to use some or all of the tables in the cluster.
+    Confirm [y/N] y
+    Node successfully repurposed to no-master and no-data.
 
-The sequence of operations for using this tool are as follows:
+.. _crate-node-cli-re-purposing-nodes-unsafe-bootstrapping:
 
-* Make sure you have really lost access to at least half of the master-eligible
-  nodes in the cluster, and they cannot be repaired or recovered by moving their
-  data paths to healthy hardware.
+Unsafely bootstrapping a cluster – ``crate-node`` unsafe-bootstrap
+------------------------------------------------------------------
 
-* Stop all remaining nodes.
+If there is at least one remaining master-eligible node, but it is not possible
+to restart a majority of the nodes, you can unsafely override the cluster’s
+voting configuration as if performing another cluster-bootstrapping process.
 
-* Choose one of the remaining master-eligible nodes to become the new elected
-  master as described above.
+If you have a choice of nodes on which to run this tool, then choose
+the one with the largest term. If there is more than one node
+with the same term, choose the one with the highest version.
 
-* On this node, run the ``crate-node unsafe-bootstrap`` command as shown
-  below. Verify that the tool reported Master node was successfully bootstrapped.
+.. NOTE::
 
-* Start this node and verify that it is elected as the master node.
+  This information identifies the node with the freshest cluster state, which
+  minimizes the quantity of data that might be lost.
 
-* Run the ``crate-node detach-cluster`` tool, described below, on every other
-  node in the cluster.
+Example
```

Comment:
the order of the example/warning subsections is different for this section and the next section. does it make sense to standardize the structure here?

Reply:
I standardized the entire document. Also I added new subheadings to the "repurpose" command section. I exchanged the word "tool" for "command" to streamline the doc. with the crate-command docs in the reference. I have rewritten many paragraphs,  structured notes and warnings and streamlined the syntax for for all admonitions.